test/integration/BUILD

```diff
@@ -26,12 +26,50 @@ skiko_runtime_platform = select({
      "@vaticle_dependencies//util/platform:is_windows": ["@maven//:org_jetbrains_skiko_skiko_awt_runtime_windows_x64"],
      "//conditions:default": ["INVALID"]})
 
+compose_test_libraries = [
+    "test-utils",
+
+    "@maven//:junit_junit",
+    "@maven//:org_jetbrains_compose_foundation_foundation_desktop",
+    "@maven//:org_jetbrains_compose_foundation_foundation_layout_desktop",
+    "@maven//:org_jetbrains_compose_material_material_desktop",
+    "@maven//:org_jetbrains_compose_runtime_runtime_desktop",
```

Comment:
Think we're specifying some dependencies twice here?
`@maven//:org_jetbrains_compose_runtime_runtime_desktop` and `@maven//:junit_junit` are specified in `deps` of `test-utils`.

But maybe that's required by Bazel because we don't automatically fetch transitive dependencies?

Have we tried removing the duplicate entries?

Reply:
We definitely need junit_junit within each test class as we decorate all our tests with `@Test`. 

We were able to cut the lines you highlighted however.